Update core-services.md (#738)
cap-js/docs: node.js/core-services.md

function srv.prepend(()=>{...})
```

> Sometimes, you need to register handlers to run before handlers registered by others before. Use `srv.prepend()` to do so, for example like this:

Reviewer: ```suggestion
If you need to register a handler that has to run before the existing ones, use `srv.prepend()` to do so, for example like this:
```
Would this be fine? @johannes-vogel

Author: That sentence is clearer, thanks! What still isn't clear to me: What if there are multiple prepends on the same service? Is that not possible and execution fails? If not, how is the order of execution defined then?

Author: @johannes-vogel I'll merge this as I think the sentence is better/clearer and definitely not wrong. If you want to adapt it, we an of course do so :)
